cg/models/nf_analysis.py

```diff
@@ -68,7 +68,7 @@ class FileDeliverable(BaseModel):
     path: str
     path_index: str | None
     step: str
-    tag: str
+    tag: str | None
```

Comment:
This should be a requiered value. What's the reason behind having empty tags in the deliverables file?

```suggestion
    tag: str
```

Reply:
MIP did not have defined tags for all files and I took MIP as starting point, but I added tags now removing None is fine

Reply:
Perhaps we shouldn't include those files in the deliverables, should we? Maybe I'm missing something here, but would be good to have a clean deliverables with only the files we want to store

Reply:
They are stored! Hermes just only greps on the step, but no problem, I added the tags

Reply:
🤯  good to know, thanks!